src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -23,15 +49,29 @@
 {% set _css_class = 'ecl-banner' %}
 {% set _extra_attributes = 'data-ecl-banner data-ecl-auto-init="Banner"' %}
 {% set _picture = picture|default({}) %}
-{% set _variant = variant|default('plain-background') %}
+{% set _video = video|default({}) %}
 {% set _size = size|default('m') %}
-{% set _centered = centered ?? false %}
+{% set _font_size = font_size|default('m') %}
+{% set _font_color = font_color|default('dark') %}
+{% set _background = background|default('light') %}
+{% set _horizontal = horizontal|default('left') %}
+{% set _vertical = vertical|default('center') %}
 {% set _full_width = full_width|default(false) %}
 {% set _title = title|default('') %}
 {% set _description = description|default('') %}
 {% set _image = image|default('') %}
 {% set _credit = credit|default('') %}
 {% set _link = link|default({}) %}
+{% set _sr_play = sr_play|default('') %}
+{% set _sr_pause = sr_pause|default('') %}
+{% set _icon_path = icon_path|default('') %}
+
+{# DEPRECATED #}
+{% set _variant = variant|default('') %}
```

Comment:
We still have in the docs plain-background as the default for this, both in the template and in the readme